dlt/destinations/impl/bigquery/bigquery.py

```diff
@@ -266,15 +275,23 @@ def _get_table_update_sql(
                     f" RANGE_BUCKET({self.capabilities.escape_identifier(c['name'])},"
                     " GENERATE_ARRAY(-172800000, 691200000, 86400))"
                 )
-        if cluster_list:
-            sql[0] = sql[0] + "\nCLUSTER BY " + ",".join(cluster_list)
+        if cluster_list := {
+            c["name"] for c in new_columns if c.get("cluster") or c.get(CLUSTER_HINT, False)
+        }:
+            sql[0] = sql[0] + "\nCLUSTER BY " + ", ".join(cluster_list)
         return sql
 
-    def _get_column_def_sql(self, c: TColumnSchema, table_format: TTableFormat = None) -> str:
-        name = self.capabilities.escape_identifier(c["name"])
-        return (
-            f"{name} {self.type_mapper.to_db_type(c, table_format)} {self._gen_not_null(c.get('nullable', True))}"
+    # noinspection PyTypedDict
```

Comment:
what's this? some kind of mypy hint?

Reply:
Pycharm specific flags - apologies, removed!